index.js

```diff
@@ -43,6 +43,79 @@
 //.
 //. Sanctuary is designed to work in Node.js and in ES5-compatible browsers.
 //.
+//. ## Ramda
+//.
+//. Ramda provides a number of partial functions. These necessitate the use
+//. of guards or null checks. In order to safely use `R.head`, for example,
+//. one must ensure that the array is non-empty:
+//.
+//.     if (R.isEmpty(xs)) {
+//.       // ...
+//.     } else {
+//.       return f(R.head(xs));
+//.     }
+//.
+//. Using the Maybe type renders such guards (and null checks) unnecessary.
+//. Changing functions such as `R.head` to return Maybe values was proposed
+//. in [ramda/ramda#683][], but was considered too much of a stretch for
+//. JavaScript programmers. Sanctuary was released the following month,
+//. in January 2015, as a companion library to Ramda.
+//.
+//. In addition to broadening in scope in the years since its release,
+//. Sanctuary's philosophy has diverged from Ramda's in several respects.
+//.
+//. ### Totality
+//.
+//. Every Sanctuary function is defined for every value in its domain. Ramda
```

Comment:
How about something like: *"The type of every Sanctuary function fully describes all of the possible input and output of the function"*?